Update troubleshoot-security-config-mgt.md (#8756)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/troubleshoot-security-config-mgt.md

Through the Microsoft Defender for Endpoint portal, security administrators can now troubleshoot Security Management for Microsoft Defender for Endpoint onboarding.

In **Configuration management** the **Onboarded via MDE security management** widget has been added to present the enrollment status breakdown of Microsoft Defender for Endpoint-managed devices.

> To see a list of all devices managed by Microsoft Defender for Endpoint, click on the **View all devices managed by MDE** button.

Reviewer: ```suggestion
To see a list of all devices managed by Microsoft Defender for Endpoint, select **View all devices managed by MDE**.
```

In the list, if a device's enrollment status is not "Success", select the device to see troubleshooting details in the side panel, pointing to the root cause of the error, and corresponding documentation.

:::image type="content" source="./images/secconfig-mde-error.png" alt-text="The filter criteria applied on the device inventory page" lightbox="./images/secconfig-mde-error.png":::

> [!NOTE] 

> > We are aware of an issue impacting the accurate detection of third party MDM's when trying to use the security management feature and are working on a fix. 

Reviewer: ```suggestion
> We are aware of an issue impacting the accurate detection of third-party MDMs when trying to use the security management feature and are working on a fix. 
```
